### ISSUE TITLE
A remaining deprecation warning

### ISSUE THREAD

**Issue by JockLawrie** [JuliaWeb/HTTP.jl#315]
Julia 0.7.0
HTTP.jl 0.7.0

When `using HTTP` I get this warning:
```julia
┌ Warning: Deprecated syntax `multiple line breaks between doc string and object` at /home/jock/.julia/packages/HTTP/mwR9J/src/Messages.jl:311.
│ Use `at most one line break` instead.
└ @ ~/.julia/packages/HTTP/mwR9J/src/Messages.jl:311
```

**Comment by oxinabox** [comment]
Fixed on master in #317,
this can be closed.

**Comment by oxinabox** [comment]
Can we tag a patch release for that and for the download fix?
